addons/stock_account/tests/test_stockvaluation.py

```diff
@@ -916,7 +916,7 @@ def test_fifo_perpetual_4(self):
 
         self.assertEqual(self.product1.standard_price, 16)
 
-        self.assertEqual(return_pick.move_lines.stock_valuation_layer_ids.unit_cost, 16)
+        self.assertEqual(round(return_pick.move_lines.stock_valuation_layer_ids.unit_cost, 1), 11.2)
```

Comment:
could you move this line and the previous line in the diff in their own commit? i think we'd need to backport them in v13 -_-